scripts/get_licensee.rb

```diff
@@ -0,0 +1,2 @@
+licensees = Licensee.find_by hair_color: 'brown' or hair_color: 'black' and
+    eye_color: 'Hazel'
```

Comment:
Unfortunately, this will not work - `and` binds more tightly than `or`, so the `and` will be performed first.